test/unit/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -1,13 +1,39 @@
 const expect = require('chai').expect;
 const assert = require('chai').assert;
 const { uniq, uniqWith, map, filter } = require('lodash');
+const load = require;
 
-const caseTypeTab = Object.assign(require('definitions/divorce/json/CaseTypeTab/CaseTypeTab'), {});
-const caseField = Object.assign(require('definitions/divorce/json/CaseField/CaseField'), {});
-
+let caseTypeTab = Object.assign(require('definitions/divorce/json/CaseTypeTab/CaseTypeTab'), {});
+let caseField = Object.assign(require('definitions/divorce/json/CaseField/CaseField'), {});
 const tabIds = uniq(map(caseTypeTab, 'TabID'));
 
+function loadAllFilesIn(location, files) {
```

Comment:
not bad idea with location, but maybe we should do it like `isFieldDuplicated` and then reuse it in other files?

Reply:
Yes at @yarpo, tried that but soon as i did that require began to act up and could not find the json files. I'll have a look before this ticket is complete to move this to a reusable location and make sure if finds the right files

Reply:
I meant soemthing like this:

```function loadAllFilesInLocation(location) {
    return loadAllFiles(files) {
		let definitions = [];

		files.forEach(file => {
			definitions = definitions
		    	.concat(load(`definitions/divorce/json/${location}/${file}.json`));
		});

		return definitions;
}```

and then:

```
const loadAllFiles = require('../../utils/utils').loadAllFilesInLocation('path/to/the/location');
```

Reply:
@yarpo ok, thats an idea also, let me take it for a spin.